MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -471,16 +482,34 @@ protected void seeking() {
             dispatch(new PauseEvent(null));
         }
         state = PlayerState.SEEKING;
+        seekingInProgress = true;
+        numberOfFramesRenderedSinceSeekingStarted = 0;
         dispatch(new SeekingEvent(null));
     }
 
-    protected void seeked() {
+    protected void seeked( boolean newFrameRendered ) {
         /*
          * Seeked event will be fired by the player immediately after seeking event
-         * This is not accurate, instead report the seeked event on first playing or pause
-         * event after seeked was reported by the player.
+         * This is not accurate, instead report the seeked event on first few frames rendered.
+         * This function is called each time a new frame is about to be rendered.
          */
-        state = PlayerState.SEEKED;
+        if ( seekingInProgress ) {
+            if ( newFrameRendered ) {
+                if ( numberOfFramesRenderedSinceSeekingStarted > 5 ) {
```

Comment:
Please change this to 2 frames to improve rebuffering detection:
```suggestion
                if ( numberOfFramesRenderedSinceSeekingStarted > 2 ) {
```

Reply:
Also, please make this a constant.